lighthouse-core/test/audits/dobetterweb/no-vulnerable-libraries-test.js

```diff
@@ -11,6 +11,21 @@ const assert = require('assert');
 
 /* eslint-env mocha */
 describe('Avoids front-end JavaScript libraries with known vulnerabilities', () => {
+  describe('#normalizeVersion', () => {
+    it('should leave valid and unsavable versions untouched', () => {
+      assert.equal(NoVulnerableLibrariesAudit.normalizeVersion(null), null);
+      assert.equal(NoVulnerableLibrariesAudit.normalizeVersion('52.1.13'), '52.1.13');
+      assert.equal(NoVulnerableLibrariesAudit.normalizeVersion('52.1.13-rc.1'), '52.1.13-rc.1');
+      assert.equal(NoVulnerableLibrariesAudit.normalizeVersion('c0ab71056b936'), 'c0ab71056b936');
+    })
+
+    it('should fix bad version numbers', () => {
+      assert.equal(NoVulnerableLibrariesAudit.normalizeVersion('11.51'), '11.51.0');
```

Comment:
we want it to do this?

Reply:
seemed reasonable to me as there were a lot in sentry, but I'm not sure any libraries that we for which we get just `major.minor` actually have a corresponding npm package

I'm fine to revert if you thing its bad

Reply:
nah its fine